src/Service.Tests/SqlTests/SqlTestBase.cs

```diff
@@ -70,16 +70,24 @@ public abstract class SqlTestBase
         /// <param name="context"></param>
         /// <param name="customQueries">Test specific queries to be executed on database.</param>
         /// <param name="customEntities">Test specific entities to be added to database.</param>
+        /// <param name="isRestBodyStrict">Whether request body for REST allows extraneous fields.</param>
         /// <returns></returns>
         protected async static Task InitializeTestFixture(
             TestContext context,
             List<string> customQueries = null,
-            List<string[]> customEntities = null)
+            List<string[]> customEntities = null,
+            bool isRestBodyStrict = true)
         {
             TestHelper.SetupDatabaseEnvironment(DatabaseEngine);
             // Get the base config file from disk
             RuntimeConfig runtimeConfig = SqlTestHelper.SetupRuntimeConfig();
 
+            // Setting the rest request body strict flag as per the test fixtures.
+            if (!isRestBodyStrict)
```

Comment:
Why do we need this if check?
Cant we just set it to whatever the function argument's value is ?

Reply:
We can. I just did this to avoid unnecessarily using `with` multiple times.